docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -124,7 +142,7 @@ If you met error when importing `intel_extension_for_pytorch`, please ensure tha
   conda install libuv
   ```
 
-* Make sure you have configured oneAPI environment variables in your Anaconda Prompt through
+* For oneAPI installed using Offline installer, make sure you have configured oneAPI environment variables in your Anaconda Prompt through
```

Comment:
Also update this in `Runtime Configuration section`?
![image](https://github.com/intel-analytics/BigDL/assets/21994046/553594df-341b-481d-9a77-3233e2656a75)